boards/bastwan/Makefile.dep

```diff
@@ -0,0 +1,7 @@
+ifneq (,$(filter netdev_default,$(USEMODULE)))
+  USEMODULE += sx1276
+endif
+
+ifneq (,$(filter saul_default,$(USEMODULE)))
+  USEMODULE += saul_gpio
+endif
```

Comment:
will test uf2 later and submit new pr

Reply:
Ok, but USB STDIO might be an easy addition - what do you think?

Reply:
I see the point that it might be easier for one to use cdc acm usb for stdio and debugging. I included your suggestion in my local build and did some power measurements. With cdc acm usb enabled the module power consumption increases about 40uA. Might be my fault somewhere but a qiuck look did not reveal a reason why the power consumption increases. 

Maybe you can give me a hint how to disable cdc acm in Makefile of project so that I can push the changes and have cdc acm disabled on demand? I am a starter in riot build and Makefiles :-)

Reply:
Oh that's a good point!
It should be possible to only clock the USB peripheral if USB is plugged it, but this has not been implemented yet.
If you add an alternative stdio method in your application Makefile, `stdio_cdc_acm` will not be used.

e.g.

    USEMODULE += stdio_uart

Reply:
Works like a charm. Added `stdio_cdc_acm` in 76c0f028c0b51ec2932a1be0e4a6c2d22e5e0680